nbfetch/handlers.py

```diff
@@ -345,40 +343,53 @@ def login(self):
     @web.authenticated
     @gen.coroutine
     def get(self):
+        """
+        Get or go to the required resource. Parameters:
+        id - Resource id (required)
+        start - notebook to launch (optional)
+        app - Optinal. 'lab' will try to run jupyter lab
+        overwrite - Overwrite any existing local copy or the resource
+        goto - Do not overwrite.  Just go to the resouce 
+        """
+
         app_env = 'notebook'
         self.log.info('HS GET ' + str(self.request.uri))
 
         self.login()
 
         id = self.get_argument('id')
-        urlPath = self.get_argument('urlpath', None) or \
-                  self.get_argument('urlPath', None)
         start = self.get_argument('start', '')
         app = self.get_argument('app', app_env)
-        
-        
+        overwrite = self.get_argument('overwrite', 0)
+        goto = self.get_argument('goto', 0)
 
-        # create Downloads if necessary
+        print('GET', id , start, app, overwrite, goto)
```

Comment:
If this is a useful statement to log out could you put in a logger?

Reply:
Good idea. Changed to self.log.info()